roles/node_images_base/files/scripts/metal/create-kis-artifacts.sh

```diff
@@ -85,6 +86,8 @@ if [[ "$1" != "squashfs-only" ]]; then
 fi
 
 if [[ "$1" != "kernel-initrd-only" ]]; then
+  echo "Removing character special files from the filesystem"
+  find /mnt/squashfs -type c -exec rm -f {} \;
```

Comment:
Maybe put single quotes around the brackets so the shell doesn't try to interpret them?
`find /mnt/squashfs -type c -exec rm -f '{}' \;`